src/result/result.py

```diff
@@ -505,11 +505,17 @@ def is_ok(result: Result[T, E]) -> TypeGuard[Ok[T]]:
     """A typeguard to check if a result is an Ok
 
     Usage:
-    >>> r: Result[int, str] = get_a_result()
-    >>> if is_ok(r):
-    >>>     r   # r is of type Ok[int]
-    >>> elif is_err(r):
-    >>>     r   # r is of type Err[str]
+
+    ```python
+
+    r: Result[int, str] = get_a_result()
+    if is_ok(r):
+        r   # r is of type Ok[int]
+    elif is_err(r):
+        r   # r is of type Err[str]
+
+    ```
```

Comment:
The codeblocks shouldn't have empty lines before and after them, they should be like this,

<pre><code>```python
r: Result[int, str] = get_a_result()
if is_ok(r):
    r   # r is of type Ok[int]
elif is_err(r):
    r   # r is of type Err[str]
```
</code></pre>